src/OrchardCore.Modules/OrchardCore.Rules/Drivers/ContentTypeConditionEvaluatorDriver.cs

```diff
@@ -31,10 +32,10 @@ public ContentTypeConditionEvaluatorDriver(IConditionOperatorResolver operatorRe
         public override Task<IDisplayResult> DisplayAsync(ContentItem contentItem, BuildDisplayContext context)
         {
             // Do not include Widgets or any display type other than detail.
-            if (context.DisplayType == "Detail" && context.Shape.TryGetProperty(nameof(ContentTypeSettings.Stereotype), out string stereotype))
+            if (String.Equals("Detail", context.DisplayType, StringComparison.OrdinalIgnoreCase) && !context.Shape.TryGetProperty(nameof(ContentTypeSettings.Stereotype), out string stereotype))
```

Comment:
Here that's fine to still use `context.DisplayType == "Detail"`, here `String.Equals()` will do the same wih one more method call, this to check the comparison type, and to do the same if it is `OrdinalIgnoreCase`

Reply:
And if `out string stereotype` is not used, you coul just use the `_` discard char

Reply:
Good idea @jtkech Done.